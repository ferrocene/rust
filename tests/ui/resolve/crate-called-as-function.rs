//@ revisions: edition2015 edition2018
//@ [edition2015] edition: 2015
//@ [edition2018] edition: 2018..

fn main() {
    ::foo()
    //[edition2015]~^ ERROR cannot find external crate `foo` in the crate root
    //[edition2018]~^^ ERROR cannot find crate `foo` in the list of imported crates
}
