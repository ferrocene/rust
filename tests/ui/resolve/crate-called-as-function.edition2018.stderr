error[E0425]: cannot find crate `foo` in the list of imported crates
  --> $DIR/crate-called-as-function.rs:6:7
   |
LL |     ::foo()
   |       ^^^ not found in the list of imported crates

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0425`.
